sumologic/config/user.properties

```diff
@@ -0,0 +1,4 @@
+name = {{cfg.collector_name}}
+accessid = {{cfg.accessid}}
+accesskey = {{cfg.accesskey}}
+syncSources = {{pkg.svc_files_path}}
```

Comment:
We need to add a config for ' ephemeral' as well.

Should we add other user.properties into this list as well?  While the above + emphemeral are sufficient for us for now, we'll need to expand the config for broader usage. I guess it's ok to defer adding more until we kick the tires on the base package.